### ISSUE TITLE
Symfony 4 config deprecation

### ISSUE THREAD

**Issue by ro0NL** [mmoreram/GearmanBundle#190]
Hi,

This bundle triggers

```
The "Symfony\Component\Config\Definition\Builder\TreeBuilder::root()" method called for the "gearman" configuration is deprecated since Symfony 4.3, pass the root name to the constructor instead
```

See also https://github.com/mmoreram/GearmanBundle/blob/a394c8d19295967caa411fbceb7ee6bf7a15fc18/DependencyInjection/Configuration.php#L34-L35

**Comment by daum** [comment]
@ro0NL  Thanks for bringing that to my attention.  I've made a fix for it in master and release v4.1.1 (https://github.com/mmoreram/GearmanBundle/releases/tag/v4.1.1)

**Comment by ro0NL** [comment]
Thanks @daum , much appreciated :)
